src/content/docs/kubernetes-pixie/auto-telemetry-pixie/manage-pixie-data.mdx

```diff
@@ -13,7 +13,7 @@ redirects:
   - /docs/auto-telemetry-pixie/manage-pixie-data
 ---
 
-You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce or exclude data, either by excluding specific namespaces or pods, or by collecting data for only the nodes you want.
+You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce, restrict, or exclude data, by excluding specific namespaces or pods, by collecting data for only the nodes you want, or by redacting columns that contain sensitive information.
```

Comment:
33 word sentence is grammatically correct but would be easier to skim if you changed it to bullets.
...exclude data, by:
* excluding specific...
* collecting data only...
* redacting columns...